web-example/craco.config.js

```diff
@@ -1,19 +1,18 @@
-const path = require('path');
-const webpack = require('webpack');
-const NodePolyfillPlugin = require("node-polyfill-webpack-plugin")
```

Comment:
why the changes from `'` to `"`? across all other dock repos we use single quotes, best to keep consistent i think